model-engine/model_engine_server/inference/vllm/build_and_upload_image.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Build and push vLLM docker image to ECR.
+
+set -eo pipefail
+
+if [ -z "$1" ]; then
+  echo "Must supply the image tag"
+  exit 1;
+fi
+
+IMAGE_TAG=$1
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 692474966980.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
Internal stuff?

Think we need to notify our sponsor cc @louisbarrett 🥃 🥴

Reply:
oh looks like i made change in wrong place